common/lib/xmodule/xmodule/js/spec_util.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+
+    jasmine.waitForInputAjax = function (conditionalFn) {
```

Comment:
yeah, UI Toolkit would be the perfect place and it can live in the same directory as ajax-helpers for now.